deep_quoridor/src/agents/alphazero/self_play_manager.py

```diff
@@ -107,17 +107,8 @@ def get_results(self, timeout: float = 0.1) -> Optional[list[dict]]:
                 # Merge results into one replay buffer. We sort them to make the results deterministic.
                 replay_buffer = []
                 results = sorted(self._results, key=lambda r: r.worker_id)
-                for r in results:
-                    print(f"Worker {r.worker_id} replay buffer size: {len(r.replay_buffer)}")
-                    if r.evaluator_statistics is not None:
-                        print(r.evaluator_statistics)
 
-                    # NOTE: Make sure the replay buffer size for the training agent is large enough to hold
-                    # the replay buffer results from all agents each epoch or else we'll end up discarding
-                    # some results and we'll have wasted computation by playing those games.
-                    replay_buffer.extend(r.replay_buffer)
-
-                return replay_buffer
+                return results
```

Comment:
I decided to return the raw per-worker result objects here instead of just the replay buffer so that the stats and wandb stuff could be handled in train_alphazero.py, instead of having to add it in to the SelfPlayManager, which I think is cleaner without it.